main.py

```diff
@@ -1,16 +1,36 @@
-# This is a sample Python script.
+import requests
+import numpy
+import pandas as pd
+import plotext as plt
 
-# Press Shift+F10 to execute it or replace it with your code.
-# Press Double Shift to search everywhere for classes, files, tool windows, actions, and settings.
+# requests library exercise
+url_posts = 'https://jsonplaceholder.typicode.com/posts'
+posts = requests.get(url_posts).json()
+url_users = 'https://jsonplaceholder.typicode.com/users'
+count = 0
+user_id = posts[0]['userId']
+for i in posts:
+    if i['userId'] == user_id:
+        count += 1
+# delete posts json array to save space
+del posts
+users = requests.get(url_users).json()
+print('Number of Posts by user: ' + users[user_id-1]['username'] + ' is ' + str(count))
 
+# numpy library exercise
+numbers = numpy.arange(10)
+# change the even values to 0
+numbers[numbers % 2 == 0] = 0
+print(numbers)
```

Comment:
Let's create a new file `requests_practice.py` and move this code to a function e.g. `request_posts()`. Once this is done let's call it inside of main.py. This will be an attempt to keep the code decoupled and increase readability.

Reply:
what will the commit message be like for this?

Reply:
Something like `move requests practice to a separate file`